fabassets/javascript/fabcar-backend.js

```diff
@@ -288,7 +291,7 @@ io.on('connection', socket => {
                 break;
 
             case "QUERYALL":
-                socket.emit('RESPONSE', {type: 'START', payload: `Request for QUERY All received` });
+                socket.emit('RESPONSE', {type: 'START', payload: `Request for QUERY All received from: ${req.NAME}` });
```

Comment:
req.NAME

By the way did you notice req.owner? Not sure what that gives us.

Reply:
Req.owner is the bit i added in the frontend - that sets the new field